src/main/java/frc/robot/commands/complex/DrivebyAuto.java

```diff
@@ -29,18 +31,21 @@ public class DrivebyAuto extends Command {
     double realFlywheel;
     double targetFlywheel;
     Rotation2d targetTurn;
+    double endStartTime;
```

Comment:
it is not suggested to have fields without a visibility i.e. `public` or `private` unless you specifically want the package-private behavior